README.md

```diff
@@ -23,31 +26,29 @@ $ npm install @gh-conf/gh-conf-validate
 ## Usage
 
 ```javascript
-const { validatePath } = require('@gh-conf/gh-conf-validate');
+const { validatePath } = require("@gh-conf/gh-conf-validate");
```

Comment:
Have you change these intentionally? Or your edited did it?
We use single codes for strings

Reply:
Sorry yes my editor auto-formats these, will correct asap :+1: